tests/phpunit/admin/class-wc-rest-stripe-exit-survey-controller-test.php

```
@@ -0,0 +1,86 @@
+		$response = $this->controller->dismiss( $request );
+
+		$this->assertSame( 200, $response->get_status() );
+		$this->assertTrue( $response->get_data()['success'] );
+
+		$stored = get_option( WC_REST_Stripe_Exit_Survey_Controller::OPTION_NAME );
+		$this->assertNotFalse( $stored );
+
+		// Verify the stored value is a valid ISO 8601 date.
+		$date = DateTime::createFromFormat( DateTime::ATOM, $stored );
+		$this->assertInstanceOf( DateTime::class, $date );
+	}
```

PHPMD (2.15.0)

[error] The method test_dismiss_persists_timestamp is not named in camelCase. (undefined)

(CamelCaseMethodName)